### PR TITLE
fixing phoenix dex trades (merge after transfers PR)

### PR DESCRIPTION
**Pull request by andrewhong5297**
part of solana dex trades overhaul

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @andrewhong5297

File: models/phoenix/schema.yml [hunk 1]
```diff
@@ -33,6 +33,9 @@ models:
       - &block_time
         name: block_time
         description: "UTC event block time of each DEX trade"
+      - &block_slot
```
Comment (Hosuke) [inline models/phoenix/schema.yml:36]: New columns introduced for `phoenix_v1.trades`

**Comment by andrewhong5297** [comment]
This one has to wait until another pr is merged, just fyi!

**Comment by andrewhong5297** [comment]
@jeff-dude @aalan3 I'm using a seed in the main project, do I need to move that seed over? or I call it as a source now or something instead?

**Comment by andrewhong5297** [comment]
> @jeff-dude @aalan3 I'm using a seed in the main project, do I need to move that seed over? or I call it as a source now or something instead?

nvm got it

**Review by 0xRobin — APPROVED** [review]

File: solana/models/_sector/dex/phoenix/phoenix_v1_trades.sql [hunk 2]
```diff
@@ -19,22 +19,29 @@
...
-            , ip.account_market as pool_id
-            , ip.call_tx_id as init_tx
-        FROM {{ source('phoenix_v1_solana','phoenix_v1_call_InitializeMarket') }} ip
-        LEFT JOIN {{ ref('tokens_solana_fungible') }}  tkA ON tkA.token_mint_address = ip.account_baseMint
-        LEFT JOIN {{ ref('tokens_solana_fungible') }}  tkB ON tkB.token_mint_address = ip.account_quoteMint
+  market_metadata as (
+    SELECT 
+        market_id
+        , cast(raw_base_units_per_base_unit as int) as raw_base_units_per_base_unit
+    FROM {{ ref('phoenix_solana_market_metadata') }}
```
[the remark this answers is not in the document]
Comment (0xRobin) [reply solana/models/_sector/dex/phoenix/phoenix_v1_trades.sql:26]: ah, forgot we aren't materializing the seeds in prod. 
From the comment at L152:

> (l.tokenA_filled*coalesce(mm.raw_base_units_per_base_unit,1)) --base unit can be adjusted by phoenix, i.e. for BONK it starts at 1e6. There is a script for updating the markets seed file.

@andrewhong5297 can you convert this to a values model as we do for other static datasets?

**Pull request merged**